src/expressions/operator-expr.md

```diff
@@ -70,6 +70,50 @@ let a = && && mut 10;
 let a = & & & & mut 10;
 ```
 
+### Raw address-of operators
+
+Related to the borrow operators are the *raw address-of operators*, which do not have first-class syntax, but are exposed via the macros `ptr::addr_of!(expr)` and `ptr::addr_of_mut!(expr)`.
+Like with `&`/`&mut`, the expression `expr` is evaluated in place expression context.
```

Comment:
It is more clear calling it `expr`, but if you don't name the operand, I will have to when I go through this chapter unifying the style. Your choice if you want to leave it like this or change it.

Reply:
I don't know what you mean -- the operand, `expr`, is named, isn't it?